agent/statemanager/state_manager.go

```diff
@@ -52,7 +52,9 @@ const (
 	// 7)
 	//   a) Add 'MetadataUpdated' field to 'api.Container'
 	//   b) Add 'DomainNameServers' and 'DomainNameSearchList' in `api.ENI`
-	ECSDataVersion = 7
+	// 8)
+	//   a) Added task cgroup related fields to 'api.Task'
```

Comment:
Can you please list these fields as well?

Reply:
I've resolved this later on where I've explicitly mentioned all the new fields.